README.md

```diff
@@ -205,6 +205,24 @@ Usage: cewl [OPTIONS] ... <url>
     <url>: The site to spider.
 ```
 
+### Running CeWL in a Docker container
+
+To quickly use CeWL on your machine with Docker, you have to build it :
+1. Build the container :
+    ```sh
+    sudo docker build -t cewl .
+    ```
+2. Container usage without interacting with local files :
+    ```sh
+    sudo docker run -it --rm cewl [OPTIONS] ... <url>
+    ```
+3. Container usage with local files as input or output :
+    ```sh
+    # you have to mount the current directory when calling the container 
+    sudo docker run -it --rm -v "${PWD}:/home" cewl [OPTIONS] ... <url>
```

Comment:
```suggestion
    sudo docker run -it --rm -v "${PWD}:/host" cewl [OPTIONS] ... <url>
```
Avoid overwriting /home as it could conflict with local files, break host files etc.. :)

Reply:
Hey hello ! 
Thanks for taking my PR :) This is a good point, and would also mean modifying the WORKDIR clause of the Dockerfile. I'm on it